src/modal_for_noobs/auth_manager.py

```
@@ -259,6 +260,133 @@
+            # Apply to environment and save
+            self.apply_auth_to_env(config)
+            self.save_auth(config)
+            
+            return {
+                "authenticated": True,
+                "source": "token_flow",
+                "workspace": workspace,
+                "token_id": token_id
+            }
+            
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)